aries_vcx/src/indy/proofs/prover/prover.rs

```diff
@@ -65,11 +65,13 @@ pub async fn generate_indy_proof(
         build_cred_defs_json_prover(wallet_handle, pool_handle, &credentials_identifiers)
         .await?;
 
+    let master_secret = settings::get_master_secret(&wallet_handle)?;
```

Comment:
- It seems that this is the only place where `master_secret_alias` is being read. Creating the global storage made the reading the `master_secret_alias` somewhat convenient this way, but global variables are always taxing codebase eventually.
- I suggest we remove `get_master_secret`, `set_master_secret`, and instead again just expand the signature of `generate_indy_proof` such that it must be injected by the caller of the method. 
   - In practice, this method is being called from method `build_presentation` in `protocols/proof_presentation/prover/states/presentation_request_received` and there's a few levels of calls, so this can be a bit cumbersome, you will have to add the parameter through number of methods.
   
This way `aries-vcx` will stay generic and will provide freedom in choice of `master_secret_alias`. To maintain the functionality, from `libvcx` you can for now simply keep injecting the original `DEFAULT_LINK_SECRET_ALIAS`